src/routes/auth/tokens.js

```diff
@@ -121,6 +148,25 @@ module.exports = async (server, options) => {
             return h.response().code(204);
         }
     });
+
+    // GET /v3/auth/token-scopes
+    server.route({
+        method: 'GET',
+        path: '/token-scopes',
+        options: {
+            tags: ['api'],
+            description: 'Get list of valid token scopes',
+            auth: {
+                scope: ['auth', 'all']
+            }
+        },
+        async handler(request, h) {
+            const scopes = Array.from(server.app.scopes.values());
```

Comment:
```suggestion
            const scopes = Array.from(server.app.scopes);
```
Works without the `values` for Sets. 
Another option would be `[...server.app.scopes]`  but I prefer the expressiveness of `Array.from`

Reply:
ah nice.